data/formats-data.js

```diff
@@ -4558,7 +4558,7 @@ exports.BattleFormatsData = {
 		tier: "RU"
 	},
 	elgyem: {
-		randomBattleMoves: ["nastyplot","psychic","thunderbolt","hiddenpowerfighting","substitute","calmmind","recover","trick","trickroom","signalbeam"],
+		randomBattleMoves: ["nastyplot","psychic","thunderbolt","hiddenpowerfighting","recover","trick","trickroom","signalbeam"],
```

Comment:
I don't know if it should keep Trick. It will always get Specs because how slow it is. NP/TR with Eviolite are good sets.